File: example/src/main/java/ru/virgil/example/truck/TruckDto.java
package ru.virgil.example.truck;

import lombok.Data;
import lombok.EqualsAndHashCode;
import ru.virgil.example.system.IdentifiedDto;

@EqualsAndHashCode(callSuper = true)
@Data
public class TruckDto extends IdentifiedDto {

    private int boxesCount;

}

File: example/src/main/java/ru/virgil/example/truck/TruckDto.kt
package ru.virgil.example.truck

import ru.virgil.example.system.dto.IdentifiedDto
import java.time.LocalDateTime
import java.util.*

data class TruckDto(
    override var createdAt: LocalDateTime?,
    override var updatedAt: LocalDateTime?,
    override var uuid: UUID?,
    var boxesCount: Int?,
) : IdentifiedDto
